inc/saiacl.h

```diff
@@ -927,9 +927,18 @@ typedef enum _sai_acl_table_attr_t
     SAI_ACL_TABLE_ATTR_FIELD_ACL_RANGE_TYPE,
 
     /**
+     * @brief IPv6 Next Header
```

Comment:
clarify on this "the next header in IPV6 header not the next header in the extension headers."